NewsApp/NewsApp/View/NewsCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  NewsCell.swift
+//  NewsApp
+//
+//  Created by Josselin DUBOC on 18/03/2020.
+//  Copyright © 2020 Josselin DUBOC. All rights reserved.
+//
+
+import UIKit
+
+class NewsCell: UITableViewCell {
```

Comment:
Cette classe n'est pas utilisée ?

Reply:
Non pas encore, elle le sera quand j'afficherai les news

Reply:
Tu peux mettre un TODO dedans pour dire pourquoi elle a été créé et à quoi elle servira prochainement dedans alors ;)

Reply:
done

Reply:
C'est préférable de justifier pourquoi telle classe a été créée, si elle est pas utilisée mettre un TODO ou un commentaire. Dans le cas ou tu as d'autres devs avec toi ça leur permet de comprendre quand ils vont pull la branche de dev.

Reply:
resolved in "ajout TODO"